examples/multi_label/classifier_chain.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
```

Comment:
I will work on adding an example with plots. Also please see my comment below regarding the README.txt file.